lib/src/common.dart

```diff
@@ -9,23 +9,16 @@ import 'package:crypto/crypto.dart';
 class Common {
   const Common();
 
-  // Temporary directory
-  static final path = Directory.systemTemp.path;
-
-  // Creates a store
-  static final store = newHiveDefaultVaultStore(path: path);
-
-  // Creates a vault from the previously created store
-  static final vault = store.vault<String>(
-    name: 'cookieVault',
-    eventListenerMode: EventListenerMode.synchronous,
-  );
+  // The vault containing the cookies. Can be initialized before usage using `setCookieVault`,
+  // otherwise it will be created using `createDefaultCookieVault`.
```

Comment:
```suggestion
  /// The vault containing the cookies. Can be initialized before usage using
  /// [setCookieVault], otherwise it will be created
  /// using [createDefaultCookieVault].
```

The convention in dart is to highlight variable, method, parameters, or type names in square brackets.
Backticks are used is to mark concrete values like `true` or `0`.